src/test/com/yetanalytics/squuid_test.cljc

```diff
@@ -58,3 +60,9 @@
                       [new-u max-2]))
                   (repeatedly 30)
                   (every? (partial apply =))))))))
+
+(deftest uuid->time-test
+  (testing
+   (is (let [id (squuid/generate-squuid*)
+             time (squuid/uuid->time (:squuid id))]
+         (= (:timestamp id) time)))))
```

Comment:
Some more test case suggestions:
- Testing `(= ts (-> ts squuid/time->uuid squuid/uuid->time))` to demonstrate that `time->uuid` and `uuid->time` are in some respect "inverses" of each other.
- Some tests with explicit timestamp and UUID examples, e.g. `(is (= #inst "1970-..." (squuid/uuid->time #uuid "0000000-0000-..."))`. (The same can be said for testing `squuid/time->uuid` for that matter.) This would not only serve as testing, but also as documentation to any devs looking at the tests.
- Some tests with non-UUID inputs, to ensure that you get an exception. (Again, the same can be said for `squuid/time->uuid`.)